decidim-core/app/controllers/decidim/errors_controller.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  class ErrorsController < ApplicationController
+    authorize_resource :error_pages, class: false
```

Comment:
Had to finally revert `< ApplicationController` + `skip_authorization_check`, CI is implacable... :unamused:.